lib/asn1/base/node.js

```diff
@@ -599,11 +622,33 @@ Node.prototype._encodeChoice = function encodeChoice(data, reporter) {
 Node.prototype._encodePrimitive = function encodePrimitive(tag, data) {
   var state = this._baseState;
 
-  if (tag === 'octstr' || tag === 'bitstr' || tag === 'ia5str')
+  if (tag === 'bitstr')
```

Comment:
I think it came to the point where using `/str$/` regexp may be a better than many ifs.